gammapy/utils/fitting/parameter.py

```diff
@@ -179,17 +179,25 @@ def __repr__(self):
             "min={min!r}, max={max!r}, frozen={frozen!r})"
         ).format(**self.to_dict())
 
-    def to_dict(self):
-        return dict(
-            name=self.name,
-            value=self.value,
-            factor=self.factor,
-            scale=self.scale,
-            unit=self.unit.to_string("fits"),
-            min=self.min,
-            max=self.max,
-            frozen=self.frozen,
-        )
+    def to_dict(self, do_cut=False):
```

Comment:
Suggest to change `do_cut` to this:
```
Parameters
-----------
selection : {"all", "simple"}
    Selection of information to include.
```

To me "do_cut" is not intuitive, prefer "selection" over "cut".

Making this a multi-choice string option means we can add other subsets in the future while keeping backwards-compatibility, whereas if we put a bool option we'll break every caller if we want another subset in the future. Not sure if we really need this flexibility, but I would go with the selection string option.